FlorenceBML/BMLacq720/BMLacq720.xml

```diff
@@ -35,7 +36,192 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <locus from="44vb" to="52vb"/>
+                            <title>Prayer of the Virgin</title>
+                            <incipit xml:lang="gez">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ለ፩አምላክ፡ ዛቲ፡ ጸሎት፡ ዘእግዝእትነ፤ ቅድስት፡ ድንግል፡ ማርያም፡ ንጽሕት፡ ወላዲተ፡ አምላክ። 
+                                ጸሎታ፡ ወበረከታ፡ የሀሉ፡ ምስለ፡ ገብራ። <persName role="owner" ref="PRS13994KidanaMaryam">ኪዳነ፡ ማርያም።</persName> ለዓለመ፡ ዓለም፡ አሜን። ወእመሂ፡ ተንባላታዊሆን። 
+                                ወእመሂ፡ ከለዳዊያን፤ አይሁድ። ወዮናናዊያን፡ ወሳምራዊያን። ወግብፃውያን፡ ወኖናፌ፡ ወኵሉ፤ ክርስቲያን። ወለኵሎሙ፡ ሰብእ፡ እኩያን። ዘርዝር፡ ምክሮሙ። </incipit>
+                            <explicit xml:lang="gez">ዝውእቱ፡ ኢየሱስ፡ ክርስቶስ፡ ወኵሎ፡ ሠራዊቶ፡ እሉ፤ እሙንቱ፡ ዘሰመይክዎሙ፡ ከመ፡ ትፈኑ፤ ሊተ፡ ተኢያኮስ፡ ቅድስት፡ ማርያም፡ ድንግል። 
+                                ከመ፡ ትዕተብ፡ ወትባርክ፡ ዘንተ፤ ማየ፤ ወዘንተ፡ ዘይተ። በዛቲ፡ ሰዓት። ወቅብዖ፡ ወአጥምቆ፡ ለገብብርከ፡ <persName role="owner" ref="PRS13994KidanaMaryam">ኪዳነ፡ ማርያም።</persName>
+                                ወይኩን፡ ሎቱ፡ ፍሥሐ፡ ወሐሤት፡ ወፈውስ፡ መድኃኒት። እምኀበ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ይእዜኒ፤ ወዘልፈኒ፡ ወለዓለመ፡ ዓለም፡ አሜን። ወአሜን፡ ለይኩን፤ ለይኩን፤ 
+                                ሊተ፡ ለገብርከ፡ <persName role="owner" ref="PRS13994KidanaMaryam">ኪዳነ፡ ማርያም።</persName>። </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i5">
+                            <locus from="52vb" to="59vb"/>
+                            <title ref="LIT4343Prayer"/>
+                            <incipit xml:lang="gez">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ፡ ጸሎት፡ ዘእግዝእትነ፡ ማርያም፡ ድንግል፤ ንጽሕት፡ ወላዲተ፡ ሕይወት፡ አንቅጸ፡ ብርሃን፤ አንቅተ፡ 
+                                ክብር፡ ወመሥተምሕርተ፡ ለኵሉ፡ ዓለም። ጸሎታ፡ ወበረከታ፡ የሃሉ፤ ምስልለ፡ ምገብራ፡ <persName role="owner" ref="PRS13994KidanaMaryam">ኪዳነ፡ ማርያም።</persName> ለዓለመ፡ ዓለም፡ 
+                                አሜን፡ ወእምዝ፡ ሰፍሐት፡ እደዊሃ። ወዓንቃዕደውት፡ ውስተ፤ ሰማይ፡ ወከልሐት፡ ኀበ፡ ፍቁር፤ ወልዳ፡ ኢየሱስ፡ ክርስቶስ፡ ትብል፡ ከመዝ። ነዓ፡ ኀቤየ፡ ኢየሱስ፡ ክርስቶስ፡ ወልታ፡ ጽድቅ፡ 
+                                እምየማን፡ ወእምፀጋም።</incipit>
+                            <explicit xml:lang="gez">ወመርሖ፡ ፍኖተ፡ መድኃኒት፤ ወሥረደ፡ ሎቱ፡ ኵሎ፡ ኃጢአቶ። ለገብርከ፡ <persName role="owner" ref="PRS13994KidanaMaryam">ኪዳነ፡ ማርያም።</persName>
+                                በዝንቱ፡ አስማቲከ፤ እዮ፤ እዮ፤ ዳኬ፡ ዳኬ፡ ለዓለመ፡ ዓለም፡ አሜን። </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i6">
+                            <locus from="59ra" to="59vb"/>
+                            <title>Unidentified text</title>
+                            <note>Unidentified text, according to <bibl><ptr target="bm:Marrassini1987manoscrittietiopici"/><citedRange unit="page">77
+                                </citedRange></bibl> with invocation of possibly magical names, that are infact names of mountains known from the Bible: 
+                                <foreign xml:lang="gez">አአትብ፡ ገጸየ፡ ውስተ፡ ርእሰ፡ ሳኒር፡ ወኄርሜን። ርእሱ፡ ከመ፡ ሰሂን። ወኅሩይ፤ ከመ፡ ቂድሮስ። በምድረ፡ ዮርዳኖስ፡ በአርሞንዒ፡ በደብር፤ ንዑስ፡ 
```

Comment:
Where can I place such a div? I did not find a statement on the correct place of it in the Guidelines.

Reply:
Guidelines https://betamasaheft.eu/Guidelines/?q=transcription&id=text-encoding

The transcription of a manuscript is placed in the `body`

```
<body>
         <div type="edition" xml:lang="gez">
            <div xml:id="mountains" type="textpart">
               <ab><pb n="59ra"/>
                  አአትብ፡ ገጸየ፡ ውስተ፡ ርእሰ፡ ሳኒር፡ ወኄርሜን። ርእሱ፡ ከመ፡ ሰሂን።  .........
               </ab>
            </div>
         </div>
      </body>
```
if you assign an `xml:id `you can use it as a target to point to


alternatively, if a work record is created, then this text goes into the work record, also `body`

Reply:
In this case we already have the `transkribusText`, so it is more tricky, since currently the elements `<text>` and `<body>` are part of the transkribusText

So if you create the bit for the edition here as shown above you should also
(1) remove `text `and `body `inside transkribusText.xml (so that it starts directly with `div `level)
(2) move the `xinclude `element pointing to `transkribusText `inside of the newly created `text ``body `in your main file BMLacq720

Alternatively, you can create a new file edition.xml and xinclude it together with the transkribus (which again has to be cleaned of text/body)

```
<text>
<body>

<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" 
                href="transkribusText.xml">
                <xi:fallback>
                    <!-- transkribus text -->
                </xi:fallback>
            </xi:include>

<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" 
                href="edition.xml">
                <xi:fallback>
                    <!-- edition text -->
                </xi:fallback>
            </xi:include>
  </body>
</text>
```